en/getting-started/install.md

```diff
@@ -20,16 +20,18 @@ The installer for Windows is a `.exe` executable, usually named `CocosCreator_vX
 
   Cocos Creator will take up approximately 1.25 GB of space on your system disk, so you can clean up your system disk space before installation.
 
-- If the installation fails, try executing the installer from the command line:
+- If there is a pop-up warning that "The required files cannot be installed because the CAB file does not have the correct digital signature. It may indicate that the CAB file is corrupted", try installing it with administrator privileges.
```

Comment:
If there is an error about the digital signature of CAB file, try installing it with administrator privileges.